main.tf

```diff
@@ -91,7 +91,7 @@ resource "google_secret_manager_secret" "secrets" {
 }
 
 resource "google_secret_manager_secret_version" "secret-version" {
-  for_each    = { for secret in var.secrets : secret.name => secret if can(secret.secret_data) }
+  for_each    = { for secret in var.secrets : secret.name => secret if can(secret.create_version) }
```

Comment:
```suggestion
  for_each    = { for secret in var.secrets : secret.name => secret if secret.create_version }
```

Reply:
I think can maybe unnecessary here since create_version is bool?

Reply:
Done!